docs/source/developer/accessibility.rst

```diff
@@ -0,0 +1,274 @@
+.. Copyright (c) Jupyter Development Team.
+.. Distributed under the terms of the Modified BSD License.
+
+Accessibility
+=============
+**A JupyterLab Developer's Guide**
```

Comment:
Is there a way to do a sub-title in RST?

I want the link to this article in the "Section Navigation" TOC on the left to say "Accessibility."

I also want the "On this page" TOC on the right to have the following outline:

- Where to start
- Best practices while developing
- Automated Regression Testing
- PR Review and Manual Testing
- Useful tools for development

When I do the following:

```
Accessibility: A JupyterLab Developer's Guide
=============================================
```

I like the way it shows up on the page, but I don't like the way the sub-title shows up in the left side TOC. Is there a way to do what I want? Or is what I'm doing now the best that I'm going to get?

Reply:
Thanks to @krassowski who showed me [the trick](https://github.com/jupyterlab/jupyterlab/pull/14426/files#diff-b812e9d851d8dea2349e3c88ce22b7f9dad035d39cbff8cce389ed9782673e92R50)!